<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PROCESADOR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RAM</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ram">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>481</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_display_seven_seg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital-7</family>
      <pointsize>78</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_display_seven_seg_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Display  7 segmentos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D050000L</family>
      <pointsize>34</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>►</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Carga</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_debug">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PC</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_b">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pc">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ir">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loading</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_12">
    <property name="geometry">
     <rect>
      <x>446</x>
      <y>490</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>480</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>■</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>20</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_url">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Url del archivo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
